web-frontend/src/pages/TransferCoins/SendCoins/SendForm.tsx

```diff
@@ -153,6 +159,28 @@ export function SendForm(props: SendFormProps) {
     <Spinner size={12} customClass="inline-block" />
   );
 
+  function getRecipient(e: string) {
```

Comment:
Shouldn't it be `set` ? 🤔 

```suggestion
  function setRecipient(e: string) {
```

Reply:
I renamed it to ```checkRecipient``` for clarity